bundle-size/github-utils.js

```diff
@@ -129,50 +127,27 @@ class GitHubUtils {
     ).includes(username);
   }
 
-  /**
-   * Convert a team slug to a team id.
-   *
-   * @param {string} teamName the team slug (`organization/team`).
-   * @return {number} the team id.
-   */
-  async getTeamId_(teamName) {
-    const cacheKey = `${CACHE_TEAM_ID_KEY}/${teamName}`;
-    const [org, teamSlug] = teamName.split('/');
-    let teamId = this.cache.get(cacheKey);
-    if (!teamId) {
-      this.log(`Cache miss for ${cacheKey}. Fetching from GitHub...`);
-      teamId = await this.github.teams
-        .getByName({org, team_slug: teamSlug})
-        .then(result => result.data.id);
-      this.log(
-        `Fetched team id ${teamId} for team ${teamName} from GitHub. Caching ` +
-          `for ${CACHE_TEAM_ID_TTL_SECONDS} seconds.`
-      );
-      this.cache.set(cacheKey, teamId, CACHE_TEAM_ID_TTL_SECONDS);
-    }
-    return teamId;
-  }
-
   /**
    * Get a list of all unique team members by team names.
    *
-   * @param {!Array<string>} teamNames names of full team slug namesto get
-   *   members of.
+   * @param {!Array<string>} teamNames names of full team slug names
+   *   ("org/team_slug") format to get members of.
```

Comment:
This is really hard to parse in the first couple tries (at least for me). Maybe something like
`@param {!Array<string>} teamNames GitHub team slug ex. "myorg/team_slug"`
"to get members of" probably doesn't add much

Reply:
Changed this. wdyt?